base/options.jl

```diff
@@ -100,3 +100,14 @@ end
 function is_file_tracked(file::Symbol)
     return ccall(:jl_is_file_tracked, Cint, (Any,), file) == 1
 end
+
+function colored_text(opts::JLOptions)
+    if !isempty(get(ENV, "FORCE_COLOR", ""))
+        color = true
+    elseif !isempty(get(ENV, "NO_COLOR", "")) && opts.color == 0  # `--color=auto`
+        color = false
+    else
+        color = (opts.color != 0) ? (opts.color == 1) : nothing
+    end
+    return color
```

Comment:
I would expect something like this: explicit color choice on command line has top priority; if omitted, we honor NO_COLOR and then FORCE_COLOR. One may argue what the order for the latter should be, in the end I am fine with either.

It seems like a major flaw to me that these two "standards" don't say anything about this kind of prioritization :-/.

```suggestion
    color = nothing
    if opts.color != 0
        color = (opts.color == 1)
    elseif haskey(ENV, "NO_COLOR")
        color = false
    elseif haskey(ENV, "FORCE_COLOR")
        color = true
    end
    return color
```